lib/quby/questionnaires/entities/questions/date_question.rb

```diff
@@ -3,57 +3,79 @@ module Questionnaires
     module Entities
       module Questions
         class DateQuestion < Question
+          POSSIBLE_COMPONENTS = %i( day month year hour minute )
+          COMPONENT_KEYS = Hash[POSSIBLE_COMPONENTS.zip %w( dd mm yyyy hh ii)]
+          COMPONENT_PLACEHOLDERS = Hash[POSSIBLE_COMPONENTS.zip %w( DD MM YYYY hh mm)]
```

Comment:
Ik zie dat de placeholders niet vertaald zijn in het nederlands. Dit in de haml doen met een I18n.t is helaas weer niet compatibel met je javascript code.